setup.py

```diff
@@ -32,6 +32,12 @@ def bool_from_environ(key: str):
 use_system_libraries = bool_from_environ("USE_SYSTEM_LIBS")
 use_cython_linetrace = bool_from_environ("CYTHON_LINETRACE")
 use_cython_annotate = bool_from_environ("CYTHON_ANNOTATE")
+# Python Limited API for stable ABI support is enabled by default.
+# Set USE_PY_LIMITED_API=0 to turn it off.
+# https://docs.python.org/3.14/c-api/stable.html#limited-c-api
+use_py_limited_api = bool_from_environ("USE_PY_LIMITED_API", default=True)
+# NOTE: this must be kept in sync with python_requires='>=3.8' below
+limited_api_min_version = "0x03080000"
```

Comment:
I think you can require >= 3.10, because 3.9 is going to be EOL this month?

Reply:
probably. I kept the current python_requires to minimize changes